### ISSUE TITLE
Plot from epoch 0 to 5000. Did not save the loss and accuracy explicitly. 

### ISSUE THREAD

**Issue by heebahsaleem** [keras-team/keras#19016]
The loss and accuracy for each epoch during the training process are not explicitly saved. The total no.of epochs is 5000. I have resumed my training from epoch 4765.  However, the history object returned by the model.fit method contains this information. Specifically, history.history['loss'] and history.history['accuracy'] provide the training loss and accuracy for each epoch.
Now, after the training is complete. I did not get the plot from epoch 0 to 5000. It shows me the plot from 4765 to 5000.

Is it possible to get the plot from epoch 0 to 5000?  I cannot train the model again as it will days again. Any solution?

**Comment by dugujiujian1999** [comment]
Are running on a notebook or something else ? Do you still have the console output log from epoch 0 to 4765 ?

**Comment by heebahsaleem** [comment]
I am using Visual studio code. 
I have console out log from 4765 to 5000 :(

**Comment by heebahsaleem** [comment]
> Are you running on a notebook or something else ? Do you still have the console output log from epoch 0 to 4765 ?

I am using Visual studio code.
I have console output log from 4765 to 5000 :(

**Comment by dugujiujian1999** [comment]
I think you need TensorBoard next time.  👍
- If your VS Code operates in a separate terminal window, the previous terminal window may still be open.
And you can check the log inside AppData\Roaming\Code\logs (If you are using plugins that log the output in this location.)

- If you have restarted the Python process at epoch 4765 and the only item you have preserved is the model's weights.
I recommend training at your earliest convenience as it will save more time.
 (If the loss & acc diagram is very important for you) 

This is a method described in [this](https://stackoverflow.com/questions/41061457/keras-how-to-save-the-training-history-attribute-of-the-history-object) post to save the training history attribute of the history object.

**Comment by dugujiujian1999** [comment]
I think you need TensorBoard next time.  👍
- If your VS Code operates in a separate terminal window, the previous terminal window may still be open.
And you can check the log inside AppData\Roaming\Code\logs (If you are using plugins that log the output in this location.)

- If you've rebooted the Python process at epoch 4765 and the sole component you've retained is the model's weights.
I recommend training at your earliest convenience as it will save more time.
 (If the loss & acc diagram is very important for you) 

This is a method described in [this](https://stackoverflow.com/questions/41061457/keras-how-to-save-the-training-history-attribute-of-the-history-object) post to save the training history attribute of the history object.

**Comment by SuryanarayanaY** [comment]
Hi @heebahsaleem ,

Can you confirm did you used same `history` to store both the training process? Did your `history.history['loss']` and `history.history['accuracy']` still showing the details from epoch 0 to 5000 after complete training ?

**Comment by heebahsaleem** [comment]
> Hi @heebahsaleem ,
> 
> Can you confirm did you used same `history` to store both the training process? If so during second training all the logs might have updated to latest details.
> 
> If you have saved the returned object in different variables like `history_1` and `history_2` then its possible to combine both history logs and plot them all.

unfortunately, i have used the same `history` every time i resumed the training.
